### DISCUSSION TITLE
Copilot is not working in Student Benifts

### DISCUSSION THREAD

**Discussion by usamabajwa86** [community/community#52464] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have recently get my verified in Student GitHub benefits, so after getting verified my Copilot is still not working

**Comment by Akash1134** [comment]
Hi @usamabajwa86 , please share more details on what exactly is happening. Are you getting any error codes?

**Comment by usamabajwa86** [comment, marked as answer]
thanks community/Community, Issue resolved

**Comment by jbschooley** [comment]
Found this thread trying to search through support. My Copilot student access is also not working :(
